WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -476,7 +476,6 @@ extension LoginEmailViewController: GIDSignInDelegate {
 
 extension LoginEmailViewController: LoginSocialErrorViewControllerDelegate {
     private func cleanupAfterSocialErrors() {
-        loginFields.username = ""
```

Comment:
I think we'd still want to do this from `retryWithEmail`... otherwise it might seem a bit weird to tap the "Try with another email" option and see that address in the text field.